fedmsg/commands/hub.py

```diff
@@ -17,11 +17,20 @@
 #
 # Authors:  Ralph Bean <rbean@redhat.com>
 #
-import fedmsg
+from gettext import gettext as _
```

Comment:
[gettext](https://docs.python.org/2/library/gettext.html) is in the standard library. I don't know exactly when it was introduced, but it's definitely 2.6+ so we're good.